.gitignore

```
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

- [ ] Opps!!!!! This file is not actually ignored by git 😃, please ensure that you ignored it properly.
```suggestion
test.md
node_modules/
```
[optional] I think `node_modules `suppose to have a `/` because it is a folder, that's the best practice. ℹ️